webapp-django/crashstats/crashstats/views.py

```diff
@@ -137,15 +137,12 @@ def report_index(request, crash_id, default_context=None):
     context['parsed_dump'] = parsed_dump
     context['bug_product_map'] = settings.BUG_PRODUCT_MAP
 
-    bugs_api = models.Bugs()
-    hits = bugs_api.get(signatures=[context['report']['signature']])['hits']
-    # bugs_api.get(signatures=...) will return all signatures associated
-    # with the bugs found, but we only want those with matching signature
-    context['bug_associations'] = [
-        x for x in hits
-        if x['signature'] == context['report']['signature']
-    ]
-    context['bug_associations'].sort(key=lambda x: x['id'], reverse=True)
+    context['bug_associations'] = list(
```

Comment:
The report view wants the list of bugs associated with this crash report's signature so we use `.filter()`.